File: app/src/main/java/org/tianjyan/pwd/service/OnPasswordGroupChangeListener.java
package org.tianjyan.pwd.service;

import org.tianjyan.pwd.model.PasswordGroup;

public interface OnPasswordGroupChangeListener {
    void onNewPasswordGroup(PasswordGroup passwordGroup);
    void onDeletePasswordGroup(String passwordGroupName);
    void onUpdateGroupName(String oldGroupName, String newGroupName);
}

File: app/src/main/kotlin/org/tianjyan/pwd/service/OnPasswordGroupChangeListener.kt
package org.tianjyan.pwd.service

import org.tianjyan.pwd.model.PasswordGroup

interface OnPasswordGroupChangeListener {
    fun onNewPasswordGroup(passwordGroup : PasswordGroup)
    fun onDeletePasswordGroup(passwordGroupName : String)
    fun onUpdateGroupName(oldGroupName : String, newGroupName : String)
}